packages/amazon-bedrock/src/bedrock-chat-language-model.test.ts

```diff
@@ -622,7 +622,7 @@ describe('doStream', () => {
       prompt: TEST_PROMPT,
     });
 
-    expect(await server.calls[0].requestBody).toStrictEqual({
+    expect(await server.calls[0].requestBodyJson).toStrictEqual({
```

Comment:
oh this must have been a misunderstanding. we need to keep the `requestBody` name for now (breaking change; also merge conflicts). I'll rename it on the `v5` branch instead.